src/api.js

```diff
@@ -7,3 +7,11 @@ export const fetchLists = (endpoint) => {
     return res.data[key];
   });
 };
+
+export const fetchSingles = (endpoint) => {
+  const key = endpoint.split("?")[0].split("/")[1].slice(0, -1);
```

Comment:
This is lovely problem solving, but there's a way cleaner way to achieve the same thing built into axios (google axios params). It's okay as it is, and it works, but taking advantage of what's available through the packages we're using would be nicer!

Reply:
Adjusted to use Axios Params instead